src/shared/components/common/pictrs-image.tsx

```diff
@@ -13,38 +13,47 @@ interface PictrsImageProps {
   nsfw?: boolean;
   iconOverlay?: boolean;
   pushup?: boolean;
+  placeholder?: boolean;
```

Comment:
This probably shouldn't be here, but in community-link, person-link, etc. It can then use the placeholder source picture, only if there is no icon given.